src/index.html

```diff
@@ -1,14 +1,42 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Enclosures</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Enclosures</h1>
-</body>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Enclosures</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+
+  <body>
+    <div id="qa-block" class="general blok">
+      <div class="middle blok">
+        <div class="within blok">
+        In programming languages, a closure is a technique for implementing lexically
+        scoped name binding in a language with first-class functions. Operationally, a
+        closure is a record storing a function together with an environment.
+        </div>
+      </div>
+    </div>
+    <div class="general blok">
+      <div class="middle blok">
+        <div class="within blok">
+        У програмуванні замиканням називають підпрограму, що виконується в
+        середовищі, яке містить одну або більше зв’язаних змінних. Підпрограма має
+        доступ до цих змінних під час виконання.
+        </div>
```

Comment:
потрібно добавти два пробіли до тексту щоб він не був на одному рівні з `div`
```suggestion
        <div class="within blok">
          У програмуванні замиканням називають підпрограму, що виконується в
          середовищі, яке містить одну або більше зв’язаних змінних. Підпрограма має
          доступ до цих змінних під час виконання.
        </div>
```